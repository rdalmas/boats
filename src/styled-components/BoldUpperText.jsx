import styled from "styled-components";

const BoldUpperText = styled.div`
  &:after {
    font-size: 1rem;
    font-weight: bold;
    line-height: 1.2;
    text-transform: uppercase;
    content: ${props => `' ${props.text}'`};
    white-space: pre;
  }
`;

export default BoldUpperText;