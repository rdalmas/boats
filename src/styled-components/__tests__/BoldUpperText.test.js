import React from 'react';
import { shallow } from "enzyme";

import BoldUpperText from '../BoldUpperText';

describe("Styled Components - BoldUpperText", () => {
  let component;
  beforeEach(() => {
    component = shallow(<BoldUpperText text="text" />);
  });
  it('should render the BoldUpperText correctly', () => {   
    expect(component).toMatchSnapshot();
  });
});