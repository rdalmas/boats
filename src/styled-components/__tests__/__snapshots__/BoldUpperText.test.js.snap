// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled Components - BoldUpperText should render the BoldUpperText correctly 1`] = `
<div
  className="sc-AxjAm fOfywv"
/>
`;
